cli/flox-rust-sdk/src/models/env_registry.rs

```diff
@@ -134,6 +140,9 @@ pub struct RegistryEntry {
     /// The list of environments that have existed at this path
     /// since the last time environments were garbage collected.
     pub envs: Vec<RegisteredEnv>,
+    /// The PIDs of current activations
+    #[serde(default)]
+    pub activations: HashSet<Pid>,
```

Comment:
do we ever rely on the order on disk or iteration order of activation pids?
given that PIDs are given out sequentially ... to a degree.

Reply:
Ok good, we use `BTree{Set,Map}`s in a couple places, i.e. when the file changes are being tracked and you dont want random reorders mess with that